admin/metabox/class-metabox.php

```diff
@@ -571,6 +571,14 @@ public function do_meta_box( $meta_field_def, $key = '' ) {
 
 		switch ( $meta_field_def['type'] ) {
 			case 'pageanalysis':
+			    $options = $this->options;
```

Comment:
I would remove this line and use `$this->options` directly instead of calling this variable.